modules/machineset-yaml-osp.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating_machinesets/creating-machineset-osp.adoc
+
+[id="machineset-yaml-osp_{context}"]
+=  Sample YAML for a MachineSet Custom Resource on {rh-openstack}
+
+This sample YAML defines a MachineSet that runs on
+{rh-openstack-first} and creates nodes that are labeled with
+`node-role.openshift.io/<node_role>: ""`
+
+In this sample, `infrastructure_ID` is the infrastructure ID label that is
+based on the cluster ID that you set when you provisioned
+the cluster, and `node_role` is the node label to add.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_ID> <1>
+    machine.openshift.io/cluster-api-machine-role: <node_role> <2>
+    machine.openshift.io/cluster-api-machine-type: <node_role> <2>
+  name: <infrastructure_ID>-<node_role> <3>
+  namespace: openshift-machine-api
+spec:
+  replicas: <number_of_replicas>
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_ID> <1>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_ID>-<node_role> <3>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_ID> <1>
+        machine.openshift.io/cluster-api-machine-role: <node_role> <2>
+        machine.openshift.io/cluster-api-machine-type: <node_role> <2>
+        machine.openshift.io/cluster-api-machineset: <infrastructure_ID>-<node_role> <3>
+    spec:
+      providerSpec:
+        value:
+          apiVersion: openstackproviderconfig.openshift.io/v1alpha1
+          cloudName: openstack
+          cloudsSecret:
+            name: openstack-cloud-credentials
+            namespace: openshift-machine-api
+          flavor: <nova_flavor>
+          image: <glance_image_name_or_location>
+          kind: OpenstackProviderSpec
+          networks:
+          - filter: {}
+            subnets:
+            - filter:
+                name: <subnet_name>
+                tags: openshiftClusterID=<infrastructure_ID>
+          securityGroups:
+          - filter: {}
+            name: <infrastructure_ID>-<node_role>
```

Comment:
Should be `<infrastructure_ID>-worker` to re-use the worker security group.

The instructions for AWS for instance re-use the worker security group.

Reply:
Thanks!

Reply:
Deferring (https://github.com/openshift/openshift-docs/pull/21896#issuecomment-641298033)